fe/fe-core/src/main/java/org/apache/doris/transaction/UpdateDbUsedDataQuotaDaemon.java

```diff
@@ -44,6 +44,10 @@ private void updateAllDatabaseUsedDataQuota() {
         List<Long> dbIdList = catalog.getDbIds();
         GlobalTransactionMgr globalTransactionMgr = catalog.getGlobalTransactionMgr();
         for (Long dbId : dbIdList) {
+            if (dbId == 0) {
```

Comment:
Not sure about it, better to judge it by `db.isInfoSchemaDb()`;

Reply:
ok